mysys/mf_format.c

```diff
@@ -36,6 +36,9 @@ char * fn_format(char * to, const char *name, const char *dir,
   DBUG_PRINT("enter",("name: %s  dir: %s  extension: %s  flag: %d",
 		       name,dir,extension,flag));
 
+  memset(buff, 0, FN_REFLEN);
```

Comment:
I can't see a case where buff would be read before written to. Can you re-run with line numbers to show where MSAN thinks the problem is?

Reply:
> I can't see a case where buff would be read before written to. Can you re-run with line numbers to show where MSAN thinks the problem is?

Our team working on it - got some problems with build with enabled line numbers. Will update as soon as we get the result